### ISSUE TITLE
Switching account fails

### ISSUE THREAD

**Issue by gordin123** [nextcloud/android#4055]
### Actual behaviour
- when switching to another account (by means of the drop down menu with the account list) the app simply stays on the currently active account.
removing the second account and adding it again will give access to the account until switching to the first account, again.

it is possible to switch accounts by selecting the icon on the top of the screen.

### Expected behaviour
- the selected account should be activated when selecting its entry from the drop down list
 
### Steps to reproduce
1. add two accounts
2. switch to the first account
3. switch to the second account

### Environment data
Android version: 8.1.0

Device model: moto g5+

Stock or customized system: stock

Nextcloud app version: 3.6.1

Nextcloud server version: 14.0.1

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by tobiasKaminsky** [comment]
This is already fixed via #4031 (there is also a APK you can test) and is testable with gplay beta version.
Maybe you want to give it a try?

**Comment by AndyScherzinger** [comment]
Please update to 3.6.2 and re-open this issue if the issue persists
